fastlane/Fastfile

```
@@ -0,0 +1,87 @@
+    # Build the debug variant
+    gradle(
+      task: "assemble#{options[:variant]}Debug"
+    )
+
+    puts "Testers group: #{ENV['TESTER_GROUPS']}"
+
+    # Distribute to Firebase App Distribution for Stag and UAT
+    firebase_app_distribution(
+      app: ENV['FIREBASE_APP_ID'],
+      firebase_cli_token: ENV['FIREBASE_CLI_TOKEN'],
+      groups: "#{ENV['TESTER_GROUPS']}",
```

rubocop (1.69.1)

[convention] Prefer `to_s` over string interpolation.

(Style/RedundantInterpolation)